app/src/main/java/org/jellyfin/androidtv/integration/dream/composable/DreamHost.kt

```diff
@@ -101,13 +103,34 @@ private suspend fun getRandomLibraryShowcase(
 			format = ImageFormat.WEBP,
 		)
 
-		val backdrop = withContext(Dispatchers.IO) {
-			imageLoader.execute(
-				request = ImageRequest.Builder(context).data(backdropUrl).build()
-			).drawable?.toBitmap()
-		} ?: return null
+		val logoUrl = api.imageApi.getItemImageUrl(
+			itemId = item.id,
+			imageType = ImageType.LOGO,
+			tag = tag,
```

Comment:
This is the tag for the backdrop, not the logo

Reply:
Is a tag required for the logo? if so does something like this seem correct?

```kotlin
val logoTag = item.parentLogoImageTag!!.randomOrNull()
     ?: item.imageTags?.get(ImageType.LOGO)
```